vllm/entrypoints/chat_utils.py

```diff
@@ -303,6 +303,28 @@ def parse_audio(self, audio_url: str) -> None:
         self._add_placeholder(placeholder)
 
 
+def validate_chat_template(chat_template: Optional[Union[Path, str]]):
```

Comment:
I remember writing this code a while back. Did it get removed at some point?

Reply:
It's pretty similar to some of the handling in `load_chat_template` (right below it) if it fails, but doesn't open the file and do the template decoding stuff!

Reply:
I see (was on mobile so didn't check below), thanks for the explanation!

Reply:
You're welcome, thanks for the quick review! 😄